custom_components/divoom_pixoo/pages/fuel.py

```diff
@@ -36,15 +36,15 @@ def fuel(pixoo, hass, page_data: dict, FONT_PICO_8, FONT_GICKO, FIVE_PIX, ELEVEN
         _LOGGER.error("Template render error: %s", e)
         return  # Stop execution if there is a template error
 
-    font_color = tuple(page_data.get('font_color', white))
-    bg_color = tuple(page_data.get('bg_color', yellow))
-    price_color = tuple(page_data.get('price_color', white))
-    title_color = tuple(page_data.get('title_color', black))
-    stripe_color = tuple(page_data.get('stripe_color', font_color))
+    font_color = eval(page_data.get('font_color', white))
+    bg_color = eval(page_data.get('bg_color', yellow))
+    price_color = eval(page_data.get('price_color', white))
+    title_color = eval(page_data.get('title_color', black))
+    stripe_color = eval(page_data.get('stripe_color', white))
```

Comment:
isn't eval pretty unsafe? I mean, doing it like this trusts unfiltered user input... I totally get what you're doing, but I'm just a liiiiiitle bit scared with the security of doing this. Maybe we could move the color rendering of the components page to a different method? (Line 141 of the sensor.py I think?)

Reply:
well in the end it's just a user input but you're right the eval() was a stopgap to get the tuple back. i think finding the solution in the sensor.py would be better. unfortunately it's already 11pm for me -.-

Reply:
Alright, I can do this if you want. Go sleep!

Reply:
all right, many thanks for the quick response

Reply:
```python 
import ast
value = "(255, 255, 255)"

result = ast.literal_eval(value)

if isinstance(result, tuple):
    pass
```

Maybe this is the solution

Reply:
Well this annoys me, as it doesn't allow users to use the "new" color strings (with just writing the color's name).

Reply:
Alright, you can check #34.